tests/test_foundry.py

```diff
@@ -242,6 +242,11 @@ def test_globus_dataframe_load():
     _delete_test_data(f)
 
 
+def test_https_upload():
+    f = Foundry(authorizers=auths)
```

Comment:
Should we have a test here?

Reply:
yes, perhaps this wasn't easily visible but I mentioned in the PR comment that I don't want to merge this until I finish the test -- I was just trying to get the code out there before our Friday sprint planning and my brief vacation, so I could get feedback sooner.

Reply:
Makes sense. Shows that I didn't read well enough :)